x/gov/migrations/v046/convert.go

```diff
@@ -117,3 +119,99 @@ func ConvertToLegacyDeposit(deposit *v1beta2.Deposit) v1beta1.Deposit {
 		Amount:     types.NewCoins(deposit.Amount...),
 	}
 }
+
+func convertToNewDeposits(oldDeps v1beta1.Deposits) v1beta2.Deposits {
+	newDeps := make([]*v1beta2.Deposit, len(oldDeps))
+	for i, oldDep := range oldDeps {
+		newDeps[i] = &v1beta2.Deposit{
+			ProposalId: oldDep.ProposalId,
+			Depositor:  oldDep.Depositor,
+			Amount:     oldDep.Amount,
+		}
+	}
+
+	return newDeps
+}
+
+func convertToNewVotes(oldVotes v1beta1.Votes) v1beta2.Votes {
+	newVotes := make([]*v1beta2.Vote, len(oldVotes))
+	for i, oldVote := range oldVotes {
+		// All oldVotes don't have the Option field anymore, as they have been
+		// migrated in the v043 package.
```

Comment:
Does it make sense to convert the Option into a `NewWeightedVoteOption` as well or do we assume that the `Option` field will be blank?

Reply:
Yeah, maybe just to be sure, let's also convert the legacy `Option` field to a weighted one.